x-pack/plugins/alerts/server/authorization/alerts_authorization_kuery.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { remove } from 'lodash';
+import { nodeTypes } from '../../../../../src/plugins/data/common';
+import { KueryNode } from '../../../../../src/plugins/data/server';
+import { RegistryAlertTypeWithAuth } from './alerts_authorization';
+
+export const is = (fieldName: string, value: string | KueryNode) =>
+  nodeTypes.function.buildNode('is', fieldName, value);
```

Comment:
@nchaulet found out that this can be further optimized with the following per https://github.com/elastic/kibana/pull/76589/files#diff-a2332bee6fd25f1c27b32e5c9997cfdeR36-R40:

```suggestion
export const is = (fieldName: string, value: string | KueryNode) =>
  nodeTypes.function.buildNodeWithArgumentNodes('is', [
    nodeTypes.literal.buildNode(fieldName),
    typeof value === 'string' ? nodeTypes.literal.buildNode(value) : value,
    nodeTypes.literal.buildNode(false),
  ]);
```

Reply:
Cool, thanks, done 👍